widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerController.kt

```diff
@@ -129,6 +138,18 @@ internal class ActivityWatcherForCallVisualizerController(
         }
     }
 
+    override fun onLink1Clicked() {
+        Logger.d(TAG, "onLink1Clicked")
+        watcher.openWebBrowserActivity(
+            stringProvider.getRemoteString(R.string.engagement_confirm_link1_text),
+            stringProvider.getRemoteString(R.string.engagement_confirm_link1_url)
+        )
+    }
+
+    override fun onLink2Clicked() {
+        TODO("Not yet implemented")
```

Comment:
To double check, this is going to be implemented later or you missed this?

Reply:
fixed 86fc6acce8746f4188120d1208f2dcde0a45632f